pkg/security/probe/probe_linux.go

```diff
@@ -753,7 +753,7 @@ func (p *Probe) handleEvent(CPU int, data []byte) {
 			pce = p.resolvers.ProcessResolver.Resolve(event.PTrace.PID, event.PTrace.PID, 0, false)
 		}
 		if pce == nil {
-			pce = model.NewEmptyProcessCacheEntry(event.PTrace.PID, event.PTrace.PID, false)
+			pce = model.GetPlaceholderProcessCacheEntry(event.PTrace.PID, event.PTrace.PID, false)
```

Comment:
We should probably keep using `NewEmptyProcessCacheEntry` here as this "target" process context will conflict with the event process cache entry in case the later fails to be resolved.